docs/best/pitfalls.md

```diff
@@ -105,7 +105,7 @@ class Profile extends React.Component {
 ### Render callbacks are _not_ part of the render method
 
 Because `observer` only applies to exactly the `render` function of the current component; passing a render callback or component to a child component doesn't become reactive automatically.
-For more details, see the [what will Mobx react to](https://github.com/mobxjs/mobx/blob/gh-pages/docs/best/react.md#mobx-only-tracks-data-accessed-for-observer-components-if-they-are-directly-accessed-by-render) guide.
+For more details, see the [what will Mobx react to](https://mobx.js.org/best/react.html) guide.
```

Comment:
Let's keep anchor there.
```suggestion
For more details, see the [what will Mobx react to](https://mobx.js.org/best/react.html#mobx-only-tracks-data-accessed-for-observer-components-if-they-are-directly-accessed-by-render) guide.
```

Reply:
@FredyC Oops. That was the first heading so I thought it was a dedicated page 😅 
The anchor has changed too so I changed it to the working one. I also noticed a couple more broken links to the same section so I updated those as well.